src/Application/Messenger/Handler/SendRequestConsumer.php

```diff
@@ -56,27 +56,36 @@ public function __invoke(SendRequest $sendRequest): void
             $this->fail($sendRequest->id, "Validation failed: {$this->validator->getReason()}");
         }
 
-        $bodyRenderedHtml = $this->twig->render("{$sendRequest->templateKey}.html.twig", $sendRequest->params);
+        // Instantiate a new Swift Message object
+        $email = new \Swift_Message();
+
+        $pathToImages = __DIR__ . '/../../../../images/';
+
+        $images['header'] = $email->embed(\Swift_Image::fromPath($pathToImages . 'TBG.jpg'));
+        $images['footer'] = $email->embed(\Swift_Image::fromPath($pathToImages . 'CCh.jpg'));
+
+        $params = array_merge($images, $sendRequest->params);
+
+        $bodyRenderedHtml = $this->twig->render("{$sendRequest->templateKey}.html.twig", $params);
         $bodyPlainText = strip_tags($bodyRenderedHtml);
 
         $config = $this->config->get($sendRequest->templateKey);
 
         // For each $p in the configured subjectParams, we need an array element with $emailData->params[$p].
-        $subjectMergeValues = array_map(fn($param) => $sendRequest->params[$param], $config->subjectParams);
+        $subjectMergeValues = array_map(fn($param) => $params[$param], $config->subjectParams);
```

Comment:
This change adds cognitive overhead we don't need: subject lines don't use images.